tests/_utils/test_prepare

```diff
@@ -276,4 +276,9 @@ function init_cluster(){
     sed -i "/relay-binlog-name/i\relay-dir: $WORK_DIR/worker2/relay_log" $WORK_DIR/source2.yaml
     dmctl_operate_source create $WORK_DIR/source1.yaml $SOURCE_ID1
     dmctl_operate_source create $WORK_DIR/source2.yaml $SOURCE_ID2
-}
\ No newline at end of file
+}
+
+function get_master_status() {
```

Comment:
there is a same func in `tests/upsteam_switch/lib.sh`, do we need to delete that one?

Reply:
Seems some variables in test_prepare need set in github action again, I keep two version of this function now.